docs/02.API-REFERENCE.md

```diff
@@ -4692,6 +4762,53 @@ jerry_create_typedarray_for_arraybuffer_sz (jerry_typedarray_type_t type_name,
 - [jerry_release_value](#jerry_release_value)
 
 
+## jerry_create_container
+
+**Summary**
+
+Create a jerry_value_t representing a given type container object.
+
+*Notes*:
+- Returned value must be freed with [jerry_release_value](#jerry_release_value)
+  when it is no longer needed.
+- This API depends on a build option (`JERRY_ES2015_BUILTIN_CONTAINER`) and can be checked
+  in runtime with the `JERRY_FEATURE_MAP, JERRY_FEATURE_SET, JERRY_FEATURE_WEAKMAP, JERRY_FEATURE_WEAKSET`
+  feature enum values.
+  see: [jerry_is_feature_enabled](#jerry_is_feature_enabled).
+- The ES2015-subset profile enables this by default.
+
+**Prototype**
+
+```c
+jerry_value_t
+jerry_create_container (const jerry_value_t *arguments_list_p,
+                        jerry_length_t arguments_list_len,
+                        jerry_container_type_t container_type);
+```
+
+- `arguments_list_p` - The arguments passed to the container constructor.
```

Comment:
What does these arguments mean? : Is this the size of the Map/set? The elements which would be inserted?